src/helpers/files.js

```diff
@@ -169,5 +182,6 @@ module.exports = {
   fetchGitRoot,
   parseGitIgnore,
   getCoverageFiles,
-  coverageFilePatterns
+  coverageFilePatterns,
+  getFilePath
```

Comment:
super nit: if possible, if things can be alphabetized, I like them alphabetized

Reply:
PRs welcome ;)